app/main.py

```diff
@@ -1,8 +1,42 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int | float,
+                 clean_mark: int | float,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: int | float,
+                 clean_power: int | float,
+                 average_rating: int | float,
+                 count_of_ratings: int | float) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> int | float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return income
+
+    def calculate_washing_price(self, car: Car) -> int | float:
+        return round(car.comfort_class * (self.clean_power - car.clean_mark)
+                     * self.average_rating / self.distance_from_city_center, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        if self.clean_power >= car.clean_mark:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, rating: int | float) -> None:
+        self.average_rating = round(
+            (self.average_rating * self.count_of_ratings + rating)
+            / (self.count_of_ratings + 1), 1)
```

Comment:
In this case do as follow
```suggestion
            / (self.count_of_ratings + 1), 
            1
        )
```